src/lights/PointLight.js

```diff
@@ -10,6 +10,7 @@ THREE.PointLight = function ( color, intensity, distance ) {
 
 	this.intensity = ( intensity !== undefined ) ? intensity : 1;
 	this.distance = ( distance !== undefined ) ? distance : 0;
+	this.decayExponent = ( decayExponent !== undefined ) ? decayExponent : 1;	// for physically correct lights, should be 2.
```

Comment:
OK. Here are my thoughts...

1. I figured this would happen. That is why, if we were going to add decay, I wanted it to be a parameter, because it would be difficult to get it to "look right".

2. The current model is not physically-based even with decay 2, because the distance term is in the numerator -- not the denominator. It was done that way to make attenuation unit-less, and to support a cut-off value. Saying it is "physical" is not intellectually honest.

3. Decay of 2 looks bad. It doesn't scale as desired. I believe we knew this all along, and that is why attenuation was implemented the way is was originally.

4. I would restore the original formula, but keep the refactoring so the formula is in one place.

5. Physically-based shading is about _shading_. Light attenuation is a separate issue, and the existing model is adequate.